client/src/components/lecture/CreateLecture.jsx

```diff
@@ -1,10 +1,37 @@
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 
 import { Modal } from 'semantic-ui-react';
 
+import { AuthenticationContext } from '../../context/authentication';
+
+import useResourceManagementControl from '../../hooks/resourceManagementControl';
+
+import { CreateLectureRequest } from '../../proto/resource-management-control_pb';
+
 import Form from '../UI/Form';
 
 const CreateLecture = props => {
+    const { user } = useContext(AuthenticationContext);
+
+    const { state, methods } = useResourceManagementControl();
+
+    const { course } = props;
+    const { createLecture } = methods;
+    const { token } = user;
+    const submitHandler = inputsValues => {
+        const request = new CreateLectureRequest();
+        request.setCourseid(course);
+        request.setName(inputsValues.name);
+        createLecture(request, token);
+    };
+
+    const { response, error } = state;
+    useEffect(() => {
+        if (response && !error) {
+            console.log(response);
```

Comment:
why u log?

Reply:
after your fix, here will be the logic for uploading video